internal/graph/loaders/processors.go

```diff
@@ -0,0 +1,386 @@
+package loaders
+
+import (
+	"time"
+
+	"github.com/lib/pq"
+	"github.com/pkg/errors"
+	"github.com/syncromatics/kafmesh/internal/graph/model"
+)
+
+type Processors struct {
+	ComponentByProcessor   *ComponentLoader
+	InputsByProcessor      *InputSliceLoader
+	JoinsByProcessor       *JoinSliceLoader
+	LookupsByProcessor     *LookupSliceLoader
+	OutputsByProcessor     *OutputSliceLoader
+	PodsByProcessor        *PodSliceLoader
+	PersistenceByProcessor *TopicLoader
+}
+
+func configureProcessors(loaders *Loaders) {
+	loader := &Processors{}
+	loaders.ProcessorLoader = loader
+
+	loader.ComponentByProcessor = &ComponentLoader{
+		wait:     100 * time.Millisecond,
```

Comment:
maybe a const to represent this repeated value across files

Reply:
agreed